### ISSUE TITLE
Add style to cssfixme service. 

### ISSUE THREAD

**Issue by karlcow** [webcompat/webcompat.com#848]
We're including a tool to fix CSS code, which will go live on https://webcompat.com/tools/cssfixme . The tool was developed in its own repo, which is https://github.com/webcompat/css-fixme/ .

This tool needs some more style/design work. Can the styling be more integrated with the rest of this site? 
This task might also involve changing/moving the text and explanations on the page, if you as a designer would like to do so.

Depends on #12
See also #847

**Comment by hallvors** [comment]
An important part here is the equivalent of this:

    pre strong{
        background: yellow;
        color:black;
    }

to highlight the style rules the too adds to the output.

**Comment by karlcow** [comment]
It needs a bit of reorg of the text, and better styling. 

What it currently looks like as of today.
![capture d ecran 2015-12-18 a 12 10 40](https://cloud.githubusercontent.com/assets/505230/11888413/7871347a-a580-11e5-885d-91bfc7df2eaf.png)

**Comment by magsout** [comment]
let's add some webcompat style on this tools

**Comment by miketaylr** [comment]
:+1:

**Comment by magsout** [comment]
@hallvors why `strong` element in your selector ? If i inspect DOM, I can't see any `<strong>` element

**Comment by magsout** [comment]
Let's close. Open new issue if we need something more..
